### ISSUE TITLE
Enhance!

### ISSUE THREAD

**Issue by lfrancis** [Threespot/aldf-wp#91]
http://live-aldf-shop.pantheonsite.io/product/aldf-womens-t-shirt/

We could probably provide some guidance to the client about helpful sizes of shop assets, the original and the zoomed version. Too small a zoom image and the feature becomes pointless, too large a zoom image and you get this:

![zoomme](https://user-images.githubusercontent.com/6942589/40324830-c0ce01a8-5d07-11e8-9cbe-d5e0ca687bc7.gif)

**Comment by lfhello** [comment]
@lfrancis Yes- Henry and I spoke about this- the ideal size is 800px square I believe ( @Henryhankdc ?)

We wanted to share various sizes with them to show what will happen if they have a larger or smaller image place. There is going to be a walk-through of the shop with ALDF.

Thanks for checking in!
